EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -212,7 +222,7 @@ public static string ComputePEPValuesForAllPSMsGeneric(List<PeptideSpectralMatch
         /// <returns></returns>
         private static string DetermineSearchType(List<PeptideSpectralMatch> psms)
         {
-            if (psms[1].DigestionParams.Protease.Name == "top-down")
+            if (psms[0].DigestionParams.Protease.Name == "top-down")
```

Comment:
Does this crash if there are no PSMs? I'm not sure this works for multiprotease, either.

Reply:
The PEP method requires a minimum of 100 non-null psms to be called.